src/app/shared/services/api/account.repo.ts

```diff
@@ -29,17 +36,14 @@ export class AccountRepo extends BaseRepo {
       passwordVerify: passwordConfirm
     });
 
-    const data = [{ AccountVO: accountVO, AccountPasswordVO: accountPasswordVO, SimpleVO: null }];
-
-    // HACK: This should be replaced with a function parameter
-    if (window.location.search.includes('createArchive')) {
-      data[0].SimpleVO = new SimpleVO({key: 'createArchive', value: true});
-    } else if (window.location.search.includes('noArchive')) {
-      data[0].SimpleVO = new SimpleVO({key: 'createArchive', value: false});
-    }
+    const data = [{
+      AccountVO: accountVO,
+      AccountPasswordVO:
+      accountPasswordVO,
```

Comment:
Extreme nitpick here, but why is this broken into two lines, when the other key / value pairs here are all on one line?
```suggestion
      AccountPasswordVO: accountPasswordVO,
```

Reply:
Because I wasn't paying enough attention and made a mistake! Thank you for catching it.